specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/security.json

```diff
@@ -734,6 +983,283 @@
             "additionalProperties": true,
             "description": "Changing set of properties depending on the alert type."
+        },
+        "RegulatoryComplianceStandardList": {
+            "description": "List of regulatory compliance standards response",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/RegulatoryComplianceStandard"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "RegulatoryComplianceStandard": {
+            "type": "object",
+            "description": "Regulatory compliance standard details and state",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Regulatory compliance standard data",
+                    "$ref": "#/definitions/RegulatoryComplianceStandardProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "RegulatoryComplianceStandardProperties": {
+		 "type": "object",
+            "description": "Regulatory compliance standard data",
+            "properties": {
+                "state": {
+                    "type": "string",
+                    "description": "Regulatory compliance standard state",
+                    "enum": ["Passed", "Failed", "Skipped", "Unsupported"],
+                    "x-ms-enum": {
+                        "name": "state",
+                        "modelAsString": true,
+                        "values": [{
+                                "value": "Passed",
+                                "description": "All supported regulatory compliance controls in the standard have a passed state"
+                            },
+                            {
+                                "value": "Failed",
+                                "description": "At least one supported regulatory compliance control in the standard has a state of failed"
+                            },
+                            {
+                                "value": "Skipped",
+                                "description": "All supported regulatory compliance controls in the standard have a state of skipped"
+                            },
+                            {
+                                "value": "Unsupported",
+                                "description": "No supported regulatory compliance data for the standard"
+                            }
+                        ]
+                    }
+                },
+                "passedControls": {
```

Comment:
For all such instances where the API is providing a summary of the number of passed, failed, skipped etc. controls, assessments etc. will it be better to include the actual resource Ids of the passedCOntrols or failedControls etc. Or actual resourceIds of passedAssessment, failedAssessment etc. 
It will easier then for the customers to double click using those ids and get more information by doing a GET. 
Right now, with the current design, they get the count which is good, but to get to the actual failed controls under a complianceStandard, they will have to get the LIST of all and then check each for the status. 
At the very least, you should provide Odata style filter query param support so that they can do $filter status eq failed.

Reply:
That is a great comment.
I dont want to add the whole resourceId list as it might be a very long one, but i am adding the Odata filter, as you have rightfully suggested.

Reply:
Great, if you can add description on what is allowed in the filter, it would help customers. Other than that, looks fine.